app/models/models.scala

```diff
@@ -12,7 +14,7 @@ case class Talk(
   url: String,
   authors: Seq[Author],
   location: String,
-  date: DateTime,
+  date: String,
```

Comment:
I think it only worked because we forgot to change this back to `DateTime`. We still get the `could not find implicit value for evidence parameter of type com.gu.scanamo.DynamoFormat[models.Talk]` error when you change it to a date.